sklearn/feature_selection/from_model.py

```diff
@@ -17,11 +18,21 @@ def _get_feature_importances(estimator):
     importances = getattr(estimator, "feature_importances_", None)
 
     if importances is None and hasattr(estimator, "coef_"):
-        if estimator.coef_.ndim == 1:
-            importances = np.abs(estimator.coef_)
+        if isinstance(estimator.coef_, np.ndarray):
+            if estimator.coef_.ndim == 1:
+                importances = np.abs(estimator.coef_)
+            else:
+                importances = np.sum(np.abs(estimator.coef_), axis=0)
+
+        elif sps.isspmatrix_csr(estimator.coef_):
+            importances = np.sum(np.abs(estimator.coef_.toarray()), axis=0)
```

Comment:
I'm not sure which function you are referring to. Do you mean abs or sum? the scipy sparse matrix has a sum, and the abs you can just apply to the ``data`` attribute.

I'm somewhat worried about substantiating a large sparse array into a dense array. It's only "n_classes" large but that could still be many. There is not really a need to densify before the sum imho. It's not a big deal but I would avoid it if we can.